README.md

```diff
@@ -60,7 +60,7 @@ sudo apt install -y \
     curl \
     wget \
     doxygen \
-    doc8 \
+    python3-doc8 \
```

Comment:
With just these libraries in a clean docker this process failed to import sphinxcontrib.spelling:

Could not import extension sphinxcontrib.spelling (exception: The 'enchant' C library was not found and maybe needs to be installed.

libenchant-2-2 (in Focal) should be added to the list of required libraries.